src/managers/GuildForumThreadManager.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const ThreadManager = require('./ThreadManager');
+const { TypeError } = require('../errors');
+const MessagePayload = require('../structures/MessagePayload');
+const { resolveAutoArchiveMaxLimit } = require('../util/Util');
+
+/**
+ * Manages API methods for threads in forum channels and stores their cache.
+ * @extends {ThreadManager}
+ */
+class GuildForumThreadManager extends ThreadManager {
+  /**
+   * The channel this Manager belongs to
+   * @name GuildForumThreadManager#channel
+   * @type {ForumChannel}
+   */
+
+  /**
+   * @typedef {BaseMessageOptions} GuildForumThreadMessageCreateOptions
+   * @property {Sticker} [stickers] The stickers to send with the message
```

Comment:
Looking at this:

https://github.com/discordjs/discord.js/blob/v13/src/structures/MessagePayload.js#L210

It's not accurate still as it accepts even ids.